packages/datadog-plugin-mocha/test/index.spec.js

```diff
@@ -94,25 +94,18 @@ describe('Plugin', () => {
         .get('/')
         .reply(200, 'OK')
 
-      const loadArguments = [['mocha', 'http']]
-
-      const isAgentlessTest = this.currentTest.parent.title === 'agentless'
-      const isEvpProxy = this.currentTest.parent.title === 'evp proxy'
-
-      if (isAgentlessTest) {
-        process.env.DD_API_KEY = 'key'
-        process.env.DD_APP_KEY = 'app-key'
-        process.env.DD_ENV = 'ci'
-        process.env.DD_SITE = 'datad0g.com'
-        process.env.DD_CIVISIBILITY_ITR_ENABLED = 1
-        loadArguments.push({ isIntelligentTestRunnerEnabled: true })
-        loadArguments.push({ experimental: { exporter: 'datadog' } })
-      } else if (isEvpProxy) {
-        loadArguments.push({ service: 'test' })
-        loadArguments.push({ experimental: { exporter: 'agent_proxy' } })
-      }
+      process.env.DD_API_KEY = 'key'
+      process.env.DD_APP_KEY = 'app-key'
+
+      const isAgentlessTest = this.currentTest.parent.title === 'reporting through agentless'
+      const isEvpProxyTest = this.currentTest.parent.title === 'reporting through evp proxy'
```

Comment:
Again, not a fan of inspecting test structure and naming as a configuration source for the tests. Better to just pass actual configs to a function to produce this.

Reply:
same as https://github.com/DataDog/dd-trace-js/pull/2584#discussion_r1055215875

Reply:
fixed in https://github.com/DataDog/dd-trace-js/pull/2584/commits/b7ffb778d8e282067ea59ba73348099523d68d9c